src/core/ledger/policies/immediate.js

```diff
@@ -23,7 +23,7 @@ export type Immediate = "IMMEDIATE";
 export type ImmediatePolicy = {|
   +policyType: Immediate,
   +budget: NonnegativeGrain,
-  +numIntervalsLookback: number,
+  +numIntervalsLookback?: number,
```

Comment:
I had this optional, but DL wanted policy fields to be required.  Maybe just have the parser set this as one if missing.